CMakeLists.txt

```diff
@@ -154,14 +154,27 @@ function (monero_set_target_no_relink target)
   endif()
 endfunction()
 
+option(STRIP_TARGETS_RELEASE "Strip symbols from targets in Release mode?" OFF)
+option(STRIP_TARGETS_DEBUG   "Strip symbols from targets in Debug mode?"   OFF)
+function (monero_set_target_strip target)
+  if (STRIP_TARGETS_RELEASE MATCHES ON)
```

Comment:
Does that trigger on ON or also 1, etc ?

Reply:
Good catch. It wouldn't.